index.html

```diff
@@ -214,6 +214,37 @@ <h3 class="font-style">Skills</h3>
             </section>
         </section>
 
+        <!-- Contact section -->
+        <footer class="footer">
+            <div class="contact-wrapper">
+                <h2 class="contact font-style">Contact me</h2>
+                <p class="contact-description font-style">If you have an application you are interested in developing, a
+                    feature
+                    that you
+                    need built or a project that needs coding. I’d love to help with it</p>
+                <form id="contact-form" action="https://formspree.io/f/xoqrojjn" method="post" class="form-wrapper">
+                    <fieldset class="form-fieldset">
+                        <div class="form-group">
+                            <input type="text" name="name" id="name" class="form-input font-style"
+                                placeholder="Your name" required maxlength="30" data-field="name">
+                        </div>
+                        <div class="form-group">
+                            <input type="email" name="email" id="email" class="form-input font-style"
+                                placeholder="Your email" required data-field="email">
+                        </div>
+                        <div class="form-group">
+                            <textarea name="message" rows="10" cols="30" id="message" class="form-input font-style"
+                                placeholder="Write your message here" maxlength="500" required
+                                data-field="message"></textarea>
+                        </div>
+                        <div class="form-group web-centered">
+                            <button type="submit" class="button button-enabled font-style contact-button">Get In
+                                Touch</button>
```

Comment:
- [ ] Good job so far with your contact form it's working fine :100: :+1: , however, your Button interactions specially for the `hover` need to be styled as Figma template, stick to the design as much as possible. :smiley: 

![Screenshot from 2022-07-05 13-39-08](https://user-images.githubusercontent.com/6615394/177344069-dcb6aab3-2974-4987-92e0-f178d6c9490b.png)

Reply:
@codecaiine Please was there a specified color for the button on the contact form for `hover`